taskboot/push.py

```diff
@@ -24,12 +24,21 @@ def push_artifacts(target, args):
     config = Configuration(args)
     assert config.has_docker_auth(), 'Missing Docker authentication'
 
-    # Setup skopeo
-    skopeo = Skopeo(
-        config.docker['registry'],
-        config.docker['username'],
-        config.docker['password'],
-    )
+    if args.push_tool == "skopeo":
+        push_tool = Skopeo(
```

Comment:
It would be cleaner if they both had the same interface: can you add a `login` method on Skopeo ?

Reply:
Easy!